llvm/test/MC/AMDGPU/gfx1013.s

```diff
@@ -1,4 +1,4 @@
-// RUN: llvm-mc -triple=amdgcn -mcpu=gfx1013 -mattr=+wavefrontsize32 -show-encoding %s | FileCheck %s
```

Comment:
This one should be testing both wave sizes

Reply:
why? I think we are generating the same code for both wave size all the times.

Reply:
And that should be checked.